### PR TITLE
fix the broken link of roadmap, refine the introduction, add the quickstart

### PR DESCRIPTION
**Pull request by FeynmanZhou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,21 @@ PorterLB is an open-source load balancer implementation designed for bare-metal
 
 ## Why PorterLB
 
-In cloud-based Kubernetes clusters, Services are usually exposed by using load balancers provided by cloud vendors. However, cloud-based load balancers are unavailable in bare-metal environments. PorterLB allows users to create LoadBalancer Services in bare-metal environments for external access, and provides the same user experience as cloud-based load balancers.
+In cloud-based Kubernetes clusters, Services are usually exposed by using load balancers provided by cloud vendors. However, cloud-based load balancers are unavailable in bare-metal environments. PorterLB allows users to create **LoadBalancer Services** in bare-metal environments for external access, and provides the same user experience as cloud-based load balancers.
 
 ## Core Features
 
-- BGP mode and Layer 2 mode
+- Network load-balancer implementation (Support BGP mode and Layer 2 mode)
```
Comment (Patrick-LuoYu) [inline README.md:16]: Load balancing in BGP mode and Layer 2 mode

**Review by Patrick-LuoYu — COMMENTED** [review]
Minor change. Please see the comment.

**Comment by FeynmanZhou** [comment]
> Minor change. Please see the comment.

Done.

**Comment by Patrick-LuoYu** [comment]
Sorry, located another issue. Please change "Bugfix" to "bugfixes" or "bug fixes".

File: README.md [hunk 2]
```diff
@@ -31,7 +37,7 @@ PorterLB has been adopted by [many companies](./ADOPTERS.md) all over the world.
 
 ## Roadmap
 
-[PorterLB Roadmap](docs/roadmap.md) lists the features and Bugfix for each milestone. If you have any new ideas, feature requests or suggestions, please submit a proposal. 
+[PorterLB Roadmap](doc/roadmap.md) lists the features and Bugfix for each milestone. If you have any new ideas, feature requests or suggestions, please submit a proposal.
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply README.md]: Fixed. Thanks for reviewing it.

**Comment by FeynmanZhou** [comment]
@zheng1 The ks-ci-bot seems on vacation : )

**Pull request merged**